demo/api/src/importer/data-streams/data-source.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum DataSources {
+    Api = "Api",
+    Bucket = "Bucket",
+    Local = "Local",
+    Checkin = "Checkin",
```

Comment:
`Checkin` is already deprecated, we don't need this.

Reply:
Hab ich entfernt